### PR TITLE
[CS] Clean elses

### PR DESCRIPTION
**Pull request by carusogabriel**
I've cleaned up some `else`s when we've already returned something, like #6889, now on `develop` branch :put_litter_in_its_place:

[`PHP-CS-Fixer`](https://github.com/FriendsOfPHP/PHP-CS-Fixer) helped me find them.

### CODE REVIEW

File: tests/Doctrine/Tests/OrmFunctionalTestCase.php
Comment (lcobucci) [inline tests/Doctrine/Tests/OrmFunctionalTestCase.php]: Although things work fine, this bit is quite unreadable... could you please modify this to at least use a ternary expression?

**Review by lcobucci — CHANGES_REQUESTED** [review]
LGTM apart from this tiny thing

File: tests/Doctrine/Tests/OrmFunctionalTestCase.php
Comment (carusogabriel) [reply tests/Doctrine/Tests/OrmFunctionalTestCase.php]: Pushing this change :man_technologist:

**Review by Majkl578 — APPROVED** [review]
LGTM, ready to merge once we are done rebasing & merging develop into master.

File: lib/Doctrine/ORM/Query/Expr.php
Comment (IonBazan) [inline lib/Doctrine/ORM/Query/Expr.php]: This can be `if`

**Comment by Ocramius** [comment]
LGTM :+1:

**Pull request merged**
